common/src/worldgen.rs

```diff
@@ -561,22 +486,20 @@ fn chunk_incident_enviro_factors(
 
     // this is a bit cursed, but I don't want to collect into a vec because perf,
     // and I can't just return an iterator because then something still references graph.
-    let (e1, t1, r1, h1, b1, f1) = i.next()?.into();
-    let (e2, t2, r2, h2, b2, f2) = i.next()?.into();
-    let (e3, t3, r3, h3, b3, f3) = i.next()?.into();
-    let (e4, t4, r4, h4, b4, f4) = i.next()?.into();
-    let (e5, t5, r5, h5, b5, f5) = i.next()?.into();
-    let (e6, t6, r6, h6, b6, f6) = i.next()?.into();
-    let (e7, t7, r7, h7, b7, f7) = i.next()?.into();
-    let (e8, t8, r8, h8, b8, f8) = i.next()?.into();
+    let (e1, t1, r1, b1) = i.next()?.into();
+    let (e2, t2, r2, b2) = i.next()?.into();
+    let (e3, t3, r3, b3) = i.next()?.into();
+    let (e4, t4, r4, b4) = i.next()?.into();
+    let (e5, t5, r5, b5) = i.next()?.into();
+    let (e6, t6, r6, b6) = i.next()?.into();
+    let (e7, t7, r7, b7) = i.next()?.into();
+    let (e8, t8, r8, b8) = i.next()?.into();
 
     Some(ChunkIncidentEnviroFactors {
-        max_elevations: [e1, e2, e3, e4, e5, e6, e7, e8],
+        elevations: [e1, e2, e3, e4, e5, e6, e7, e8],
```

Comment:
`max_elevation` is a very meaningful name, which I don't think should be changed. I think of "elevation" as "distance from the ground plane", which is not something any noise field controls. The actual meaning of this noise field is the maximum elevation at which a solid voxel should be generated.

Reply:
> The actual meaning of this noise field is the maximum elevation at which a solid voxel should be generated.

Makes sense. I've changed it back.

Reply:
I have also updated the commenting around the terrain-generating functions in worldgen.rs and terraingen.rs.